pkg/operator/controller/garden/garden/components.go

```diff
@@ -1140,6 +1156,8 @@ func (r *Reconciler) newBlackboxExporter() (blackboxexporter.Interface, error) {
 				gardenerutils.NetworkPolicyLabel(gardenerapiserver.DeploymentName, 8443): v1beta1constants.LabelNetworkPolicyAllowed,
```

Comment:
Don't we need a similar network policy label for virtual kube-apiserver?
I see that blackbox-exporter should target its external domain, so this could be either public network or the istio gateway, depending on the load-balancer.

Reply:
Nice catch!